src/NServiceBus.Testing.Fakes/OutgoingMessageExtensions.cs

```diff
@@ -40,13 +40,6 @@ public static IEnumerable<SentMessage<TMessage>> Containing<TMessage>(this IEnum
                 .Select(x => new SentMessage<TMessage>((TMessage) x.Message, x.Options));
         }
 
-        internal static IEnumerable<TimeoutMessage<TMessage>> Containing<TMessage>(this IEnumerable<TimeoutMessage<object>> timeoutMessages)
```

Comment:
TimeoutMessage is only relevant for NServiceBus.Testing. Since this is internal, it should be declared in the NServiceBus.Testing repo.